curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-and-share-about-educational-and-professional-background/685d7a6785617d165f78aa02.md

```diff
@@ -0,0 +1,98 @@
+---
+id: 685d7a6785617d165f78aa02
+title: Task 67
+challengeType: 22
+dashedName: task-67
+---
+
+<!-- REVIEW -->
+
+# --description--
+
+This is a review of the entire dialogue you just studied.
+
+# --instructions--
+
+Write the following words or phrases in the correct spot:
+
+`majored in`, `marathons`, `studies`, `learning experience`, `hands-on`, `played around`, and `background`.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`Brian: Hey, how's it going? I noticed you're relatively new here. What's your BLANK in tech?`
+
+`Sophie: Hey! Yeah, I joined the team a few months ago. I didn't study computer science at university, but I BLANK electrical engineering. It wasn't exactly tech-related, but I learned a lot about problem-solving and critical thinking.`
+
+`Brian: That's interesting! So, you didn't work on many programming projects during your studies, did you?`
+
+`Sophie: No, not really. I BLANK with coding, but most of my projects involved circuit designs and electrical systems. It was a bit different.`
+
+`Brian: I see. Did you enjoy your time there, even if it wasn't entirely tech-focused?`
+
+`Sophie: I did. It was challenging, but I met some fantastic people. We had a bunch of fun coding BLANK, although I wasn't always a very strong programmer.`
+
+`Brian: Well, we all have different journeys. I studied at a small college as well and majored in information technology. It was more BLANK, and I loved every bit of it. We didn't have to wear too many hats, but I got to build and maintain networks and manage a few small projects.`
+
+`Sophie: That sounds cool. Did you ever work on any big projects during your BLANK?`
+
+`Brian: They weren't exactly big projects. Most of them were smaller scale. But it was a great BLANK. It's fascinating to me how we all take different paths but end up in the same tech world, right?`
+
+`Sophie: Absolutely! Let's grab a coffee sometime and chat more!`
+
+## --blanks--
+
+`background`
+
+### --feedback--
+
+The experience, skills, or education someone has.
+
+---
+
+`majored in`
+
+### --feedback--
+
+Studied a main subject in college or university.
+
+---
+
+`played around`
+
+### --feedback--
+
+Tried something out in a fun or casual way.
+
+---
+
+`marathons`
```

Comment:
```suggestion
`coding marathons`
```

Reply:
Adding coding to the blank, since the definition for 'marathons' alone is different.